src/lib/components/ContentfulRichText/ContentfulRichText.svelte

```diff
@@ -14,6 +14,6 @@
   }
 </script>
 
-{#each document.content as subNode}
+{#each document.content as subNode (crypto.randomUUID())}
```

Comment:
Ideally I would grab an ID straight from from contentful to use as an ID, but there's not one directly on the `document` property. I'm open to alternatives, but this has content at least updating properly now